crowbar_framework/app/views/sanities/show.html.haml

```diff
@@ -0,0 +1,4 @@
+.row
+  .col-lg-12
+    .alert.alert-danger
+      = flash_for_sanity_checks(@errors)
```

Comment:
Why have you added `flash_for_sanity_checks` which generates html if it's just called here?